cmd/dra-example-controller/claimparametersgen.go

```diff
@@ -0,0 +1,229 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	resourceapi "k8s.io/api/resource/v1alpha2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+	"k8s.io/utils/ptr"
+
+	gpucrd "sigs.k8s.io/dra-example-driver/api/example.com/resource/gpu/v1alpha1"
+)
+
+const (
+	DriverAPIGroup = gpucrd.GroupName
+	DriverName     = gpucrd.GroupName
+)
+
+func StartClaimParametersGenerator(ctx context.Context, config *Config) error {
```

Comment:
This function starts `gpuClaimParametersInformer` but it's called `StartClaimParametersGenerator`. Should it be renamed to `StartClaimParametersInformer`?